### PR TITLE
Add amp-o2-player extension example page

### PR DESCRIPTION
**Pull request by KondratovSvyatoslav**
Added amp-o2-player example

### CODE REVIEW

**Comment by KondratovSvyatoslav** [comment]
Here is issue  #295

File: src/20_Components/amp-o2-player.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!--
+  #### Introduction
+
+  Embed o2 player in your AMP HTML files.
```
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: Do you want to add a link to your homepage?

File: src/20_Components/amp-o2-player.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+<!-- #### Basic Usage -->
+<!--
+To embed o2 player in your AMP HTML page you need to add amp-02-player tag with two mandatory parameters data-pid, and one of
+data-vid or data-bcid, other supported parameters are: data-bid, data-macro, layout (we recommend to use responsive),
+ width and height, last two define aspect ratio of the player.
```
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: Can you highlight tag and attribute names, using "``", e.g. add `amp-02-player` tag

File: src/20_Components/amp-o2-player.html (hunk 1 above)
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: Or any other additional documentation that might be valuable.

**Comment by sebastianbenz** [comment]
Great! Added a few comments. Thanks!

File: src/20_Components/amp-o2-player.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+-->
+<amp-o2-player
+        data-pid="579a115aa01b4d2db062aa85"
+        data-bcid="545db032e4b0af1a81424b48"
+        data-macros="m.playback=click"
+        layout="responsive"
+        width="480" height="270">
+</amp-o2-player>
```
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: Could you please use a different more "neutral" example video.

**Comment by sebastianbenz** [comment]
@KondratovSvyatoslav can you please take another look. Thanks!

**Comment by KondratovSvyatoslav** [comment]
Sorry for the delay, spent some time to figure out what home page to set, we will have formal docs for amp usage in our help soon and we will update this link accordingly

File: src/20_Components/amp-o2-player.html (hunk 3 above)
Comment (KondratovSvyatoslav) [inline src/20_Components/amp-o2-player.html]: Sorry, video changed

**Comment by KondratovSvyatoslav** [comment]
@sebastianbenz Could you please have a look. Thanks!

File: src/20_Components/amp-o2-player.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<!--
+  #### Introduction
+
+  Embed o2 player in your AMP HTML files. You can find player here http://on.aol.com/,
```
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: use markdown to format links:

" Embed the [o2 player](http://on.aol.com/) in your AMP HTML files. You can find more details in the documentation on [Github](https://github.com/ampproject/amphtml/blob/master/extensions/amp-o2-player/amp-o2-player.md)."

File: src/20_Components/amp-o2-player.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+<!-- #### Basic Usage -->
+<!--
+To embed o2 player in your AMP HTML page you need to add <code>amp-02-player</code> tag with two mandatory parameters <code>data-pid</code> , and one of
+<code>data-vid</code> or <code>data-bcid</code> , other supported parameters are: <code>data-bid</code> , <code>data-macro</code> , <code>layout</code> (we recommend to use responsive),
+<code>width</code> and <code>height</code> , last two define aspect ratio of the player.
```
Comment (sebastianbenz) [inline src/20_Components/amp-o2-player.html]: You can use markdown syntax to syntax highlight single words, e.g. 

```
To embed o2 player in your AMP HTML page you need to add `amp-02-player` tag with two mandatory parameters ...
```

**Comment by sebastianbenz** [comment]
Thanks - one small formatting nit, otherwise LGTM.

**Comment by KondratovSvyatoslav** [comment]
Formatting changed

**Comment by sebastianbenz** [comment]
LGTM - thanks a lot!

**Pull request merged**